kubectl_plugins/bind-service/bind-service.go

```diff
@@ -0,0 +1,104 @@
+package main
```

Comment:
@pmorie I have moved the plugins into the dir you have recommended `plugin/cmd/kubectl/...`